### PR TITLE
Add hideDocs, hideRoot and transformSchema options

### PR DESCRIPTION
**Pull request by timsuchanek**
In order to have a more clean overview of just the models, this PR adds 3 functionalities:
1. `hideDocs`: hide the docs sidebar
2. `hideRoot`: hides the root element, only possible if the sidebar is hidden, as otherwise the UI will break
3. `transformSchema`: custom transformation for the schema, useful when you want to adjust the schema before it gets drawn

### CODE REVIEW

File: src/reducers/index.ts [hunk 1]
```diff
@@ -16,7 +16,10 @@ var initialState = {
   displayOptions: {
     rootTypeId: null,
     skipRelay: true,
-    sortByAlphabet: false
+    sortByAlphabet: false,
+    hideDocs: false,
+    hideRoot: false,
+    transformSchema: null,
```
Comment (IvanGoncharov) [inline src/reducers/index.ts]: @timsuchanek Redux state should be pure-JSON and as I understand `transformSchema` is a function.

Comment (timsuchanek) [inline src/reducers/index.ts]: @IvanGoncharov that is a very good point. Do you see another way to pass in the function than with redux?

Comment (RomanGotsiy) [inline src/reducers/index.ts]: @timsuchanek I think we should pass this through React components props and trigger some action when it changes so redux knows about this.
Would you prefer to do it yourself or I can merge your PR and do it on top of your work?

**Comment by RomanGotsiy** [comment]
Everything except `transfromSchema` has been already introduced.
We're not sure about `transformSchema` so closing this.

Thanks!

**Pull request closed without merging**